src/cmds/cpuinfo.c

```diff
@@ -12,12 +12,12 @@
 
 int main(int argc, char **argv) {
 	struct cpu_info *info = get_cpu_info();
-	printf("CPU Vendor ID:              %s\n", info->vendor_id);
-	printf("CPU Base Frequency (MHz):   %u\n", info->base_freq);
-	printf("CPU Max Frequency (MHz):    %u\n", info->max_freq);
-	printf("CPU FPU Support:            %s\n", info->FPU == 1 ? "true" : "false");
-	printf("CPU MMX Support:            %s\n", info->MMX == 1 ? "true" : "false");
-	printf("CPU SSE Support:            %s\n", info->SSE == 1 ? "true" : "false");
+	printf("%-20s %s\n", "CPU Vendor ID ", info->vendor_id);
+	
+	for(int i = 0; i < CURRENT_ARCH_NUM_FEATURES; i++) {
```

Comment:
No need to iterate to `CURRENT_ARCH_NUM_FEATURES`; we can simply look at `info->feature_count`

Reply:
Wouldn't `CURRENT_ARCH_NUM_FEATURES` and `info->feature_count` be the same number?

Reply:
I think that better solution would be to define `CURRENT_ARCH_NUM_FEATURES` big enough for all architectures (so we don't need to change it every time we add check for a new feature)

Reply:
Okay, fair point. Will fix this, thanks!